main.py

```
@@ -124,11 +124,8 @@ def __init__(self, master: tk.Misc, columns: list[str]) -> None:
         self.scroll_x = ttk.Scrollbar(self, orient="horizontal", command=self.canvas.xview)
         self.inner = ttk.Frame(self.canvas)
 
-        self.inner.bind(
-            "<Configure>",
-            lambda e: self.canvas.configure(scrollregion=self.canvas.bbox("all")),
-        )
-        self.canvas.create_window((0, 0), window=self.inner, anchor="nw")
+        self.inner.bind("<Configure>", lambda e: self._update_scrollregion())
```

Ruff (0.13.1)

Unused lambda argument: `e`

(ARG005)